src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -904,16 +896,10 @@
     type: command
     short-summary: Create a load balancer.
     examples:
-        - name: Create a basic load balancer.
-          text: >
-            az network lb create -g MyResourceGroup -n MyLb
-        - name: Create a load balancer on a specific virtual network and subnet.
```

Comment:
This is an example of using specific optional parameters. Why remove it?

Reply:
Fixed. Must have been caught in delete of the first example.